behavior/plans/diff.py

```diff
@@ -1,466 +1,285 @@
 from __future__ import annotations
 
-import json
 import logging
 import os
 import shutil
-import sys
-import uuid
 from pathlib import Path
-from typing import Any
 
 import numpy as np
 import pandas as pd
 from pandas import DataFrame
 from plumbum import cli
+from tqdm import tqdm
 
 from behavior import BASE_TARGET_COLS, BENCHDB_TO_TABLES, DIFF_COLS, PLAN_NODE_NAMES
 
-# The set of columns present in all operating-unit datasets.
-COMMON_SCHEMA: list[str] = [
-    "statement_id",
-    "rid",
-    "pid",
-    "query_id",
-    "plan_node_id",
-    "left_child_plan_node_id",
-    "right_child_plan_node_id",
-    "statement_timestamp",
-    "ou_name",
-    "startup_cost",
-    "total_cost",
-    "plan_type",
-    "plan_rows",
-    "plan_width",
-    "start_time",
-    "end_time",
-    "cpu_id",
-] + BASE_TARGET_COLS
+from . import BLACKLIST_OUS, COMMON_SCHEMA, STANDARDIZE_COLUMNS
 
 logger = logging.getLogger(__name__)
 
 
-def remap_cols(ou_to_df):
-    """Canonicalize the column names and add unique identrifiers for all the original DataFrames.
+def load_csv(ou_index, csv_file):
+    """
+    Read the CSV file for a given OU.
 
     Parameters
     ----------
-    ou_to_df : dict[str, DataFrame]
-        Dictionary of raw DataFrames.
+    ou_index : int
+        Index into PLAN_NODE_NAMES that describes the particular OU this CSV file
+        corresponds to.
+
+    csv_file: Path
+        Path to the CSV file that should be read>
 
     Returns
     -------
-    dict[str, DataFrame]
-        Dictionary of remapped DataFrames.
+    Tuple[ou_index, df[targets], df[features]]
+
+    ou_index : index of the PLAN_NODE_NAMES that the dataframe coresponds to
+    df[targets] : dataframe constructed from the CSV file for differencing
+    df[features] : dataframe constructed from the CSV file of non-differencing columns
+
+    Note:
```

Comment:
```suggestion
    Notes
    -----
```

https://numpydoc.readthedocs.io/en/latest/format.html#notes